src/unleash.ts

```diff
@@ -245,6 +247,8 @@ export class Unleash extends EventEmitter {
   }
 
   async start(): Promise<void> {
+    if(this.started) return;
+    this.started = true;
```

Comment:
Oh, so this was the error. Ok, this is a great improvement. 

Did you also check the destroy() method?

Reply:
Do we want to start the client again after destroy? If not then destroy changes are not needed